src/utils/index.js

```diff
@@ -1,35 +1,40 @@
-export { default as isSingleLineString } from "./isSingleLineString";
-export { default as isStandardRule } from "./isStandardRule";
-export { default as isStandardSelector } from "./isStandardSelector";
-export { default as isStandardSyntaxProperty } from "./isStandardSyntaxProperty";
-export { default as isWhitespace } from "./isWhitespace";
-export { default as namespace } from "./namespace";
-export { default as optionsHaveException } from "./optionsHaveException";
-export { default as optionsHaveIgnored } from "./optionsHaveIgnored";
-export { parseFunctionArguments } from "./parseFunctionArguments";
-export { default as parseNestedPropRoot } from "./parseNestedPropRoot";
-export { default as parseSelector } from "./parseSelector";
-export { default as rawNodeString } from "./rawNodeString";
-export { removeEmptyLinesBefore } from "./removeEmptyLinesBefore";
-export * from "./ruleUrl";
-export { default as findOperators } from "./sassValueParser";
-export { default as whitespaceChecker } from "./whitespaceChecker";
-export { default as hasNestedSibling } from "./hasNestedSibling";
-export { default as isType } from "./isType";
-export { moduleNamespace } from "./moduleNamespace";
-export * from "./validateTypes";
+"use strict";
+
+const utils = {
+  addEmptyLineBefore: require("./addEmptyLineBefore"),
+  atRuleParamIndex: require("./atRuleParamIndex"),
+  beforeBlockString: require("./beforeBlockString"),
+  blockString: require("./blockString"),
+  declarationValueIndex: require("./declarationValueIndex"),
+  // Todo should be deleted after upgrade `stylelint` to `10`
+  eachRoot: require("./eachRoot"),
+  findCommentsInRaws: require("./findCommentsInRaws"),
+  functions: require("./functions"),
+  hasBlock: require("./hasBlock"),
+  hasEmptyLine: require("./hasEmptyLine"),
+  hasInterpolatingAmpersand: require("./hasInterpolatingAmpersand"),
+  hasNestedSibling: require("./hasNestedSibling"),
+  isInlineComment: require("./isInlineComment"),
+  isNativeCssFunction: require("./isNativeCssFunction"),
+  isSingleLineString: require("./isSingleLineString"),
+  isStandardRule: require("./isStandardRule"),
+  isStandardSelector: require("./isStandardSelector"),
+  isStandardSyntaxProperty: require("./isStandardSyntaxProperty"),
+  isType: require("./isType"),
+  isWhitespace: require("./isWhitespace"),
+  moduleNamespace: require("./moduleNamespace"),
+  namespace: require("./namespace"),
+  optionsHaveException: require("./optionsHaveException"),
+  optionsHaveIgnored: require("./optionsHaveIgnored"),
+  parseFunctionArguments: require("./parseFunctionArguments"),
+  parseNestedPropRoot: require("./parseNestedPropRoot"),
+  parseSelector: require("./parseSelector"),
+  rawNodeString: require("./rawNodeString"),
+  removeEmptyLinesBefore: require("./removeEmptyLinesBefore"),
+  ruleUrl: require("./ruleUrl"),
+  findOperators: require("./sassValueParser"),
+  validateTypes: require("./validateTypes"),
```

Comment:
@XhmikosR This exporting `validateTypes` seems wrong. The original code is below:

https://github.com/stylelint-scss/stylelint-scss/blob/41b71f39a462f298ffdec394cbeb9bad14542f50/src/utils/index.js#L35

I don't know how to rewrite `export *` to CommonJS, but exporting each function seems to resolve test failures like this:

```javascript
const { isString, isRegExp } = require("./validateTypes");

const utils = {
  // ...
  isString,
  isRegExp
};
```

Reply:
Ah true, thanks!

I'm thinking maybe I'll just drop utils/index.js.